libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -58,53 +58,6 @@ using namespace utils;
 
 namespace msl {  // this is only used for MSL
 
-std::string addMaybeUnusedToVariables(const std::string& mslShader) {
```

Comment:
pls remove <string> and <string_view> header includes too. We only included those for addMaybeUnusedToVariables

Reply:
The goal is to reduce the diff to upstream (currently tag v1.49.0), that is why I ask such a nit

Reply:
4d5cf85